README.md

```diff
@@ -171,6 +171,12 @@ Run the `destroy-cloudfoundry` pipeline.
 
 # Additional notes
 
+## Running tests locally
+
+You will need to install some dependencies to run the unit tests on your own
+machine. The most up-to-date reference for these is the Travis CI
+configuration in [`.travis.yml`](.travis.yml).
+
```

Comment:
Fair point. I've added a comment to the commit message about why `--user` is required. Not sure how we should address the documentation. Any suggestions?